README.md

```diff
@@ -20,7 +20,7 @@ Source code is included in the geos-3.3.3 directory.
 
 * On linux, if your python was installed via a package management system, make
 sure the corresponding "python-dev" package is also installed.  Otherwise, you
-may not have the python header (Python.h), which is required to build python
+may not have the python header (`Python.h`), which is required to build python
```

Comment:
This is reStructuredText not Markdown. Please use double backticks for literal quotes.